doc/src/modules/integrals/rubi/about.rst

```diff
@@ -0,0 +1,15 @@
+About Rule Based Integrator
+===========================
+
+About it
+--------
+
+A Rule based integrator nicknamed Rubi is module in SymPy, Integrals. It is an implementation of more than 6000 rules to cover a wide variety of 
+indefinite integration. Rubi utilizes a set of well defined rules which makes it smart to present the results in a more symmetric and simplified 
```

Comment:
It doesn't sound good "cover a wide variety of indefinite integration."

Reply:
what should I write instead of this ?